internal/transform/asset.go

```diff
@@ -15,16 +15,29 @@ func TransformAsset(operation xdr.Operation, operationIndex int32, transactionIn
 	operationID := toid.New(ledgerSeq, int32(transactionIndex), operationIndex).ToInt64()
 
 	opType := operation.Body.Type
-	if opType != xdr.OperationTypePayment {
+	if opType != xdr.OperationTypePayment && opType != xdr.OperationTypeManageSellOffer {
 		return AssetOutput{}, fmt.Errorf("operation of type %d cannot issue an asset (id %d)", opType, operationID)
 	}
 
-	op, ok := operation.Body.GetPaymentOp()
-	if !ok {
-		return AssetOutput{}, fmt.Errorf("could not access Payment info for this operation (id %d)", operationID)
+	op := xdr.Asset{}
```

Comment:
this variable name is a little confusing. Would prefer if it were renamed something like `asset` instead of `op`